inst/templates/ad_advs.R

```diff
@@ -44,8 +44,16 @@ bmi_for_age <- who_bmi_for_age_boys %>%
         SEX == 2 ~ "F",
         TRUE ~ NA_character_
       ),
-      AGE = AGE * 30.4375
-    )) %>%
+      AGE = round(AGE * 30.4375)
+    ) %>%
+    # Interpolate the AGE by SEX
+    group_by(SEX) %>%
```

Comment:
wouldn't it be cleaner to have this as `by_vars` argument within the `derive_interp_records()` function and then you could do all the group and ungroup in the function and for the template we just call something like `derive_interp_records(., parameter = "BMI", by_vars = exprs(SEX))`. at the moment this step is doing a lot and its not simple for users to follow.

Reply:
Done